### ISSUE TITLE
Form load time is too slow

### ISSUE THREAD

**Issue by maimoonak** [OpenSRP/opensrp-client#251]
Enketo Form load time is too slow and vaccinators find it difficult to do live data entry. 

- [ ] - Increase Form load time
- [ ] - Simplify form

**Comment by maimoonak** [comment]
@keymane : with reference to #262 and #174 and current ticket. Since we are also working on similar tickets could you please explain a little more about what you would be working on and how you are approaching to this issue? What changes you might be making. Is it different from what has been mentioned in these tickets?

**Comment by keymane** [comment]
@maimoonak yes, it may be different. Data access looks like a probable cause.
